README.md

```diff
@@ -97,6 +97,15 @@ Acquires the semaphore asynchronously. The wait operation is performed on a sepa
 
 Acquires the semaphore synchronously. The wait operation is performed on the main thread, and will block the event loop until the semaphore is acquired. Throws if an error occurs.
 
+### `Semafour.prototype.tryWaitSync()`
```

Comment:
Can you keep the sync and async versions. Check out how `post()` and `postSync()` do it in `lib/index.js`.

Reply:
My bad, I thought you longer wanted an async version

Reply:
I did implement `tryWait` the way `post` is done. That said, it feels a little bit weird to implement it that way. What's the justification for having a asynchronous version for an operation that is essentially synchronous?